articles/governance/machine-configuration/how-to/develop-custom-package/1-set-up-authoring-environment.md

```diff
@@ -24,7 +24,7 @@ Support for applying configurations through machine configuration is introduced
 
 Operating systems where the module can be installed:
 
-- Ubuntu 18 or higher
```

Comment:
FYI .. this is temporary. we are going to fix some issues in guestconfiguration module to make it work with higher version of ubuntu and latest powershell. we will update it to 18+ and powershell links later when we release the fix.